### ISSUE TITLE
Possible bug in automatic bible verse division across slides

### ISSUE THREAD

**Issue by jejbr84** [quelea-projection/Quelea#148]
I tried to use the automatic bible verse division across slides (option 'Max number of verses per slide' off, option 'Keep verses whole across slides' off), but I ran into some trouble understanding how it works. To my feeling there was too much text on one slide, so I looked into the code to figure out how it works.

The sections are created in BiblePassage::fillTextSections(). The maximum line length is determined from the maximum number of characters per line option. The maximum number of lines is also determined from that setting by dividing it by 4. My setting is 36 characters per line, so that should give a maxumimum of 9 lines. This seems somewhat arbitrary. Maybe it works for 4:3 ratio screens, but my screens have 16:9 ratio for which 9 lines is too much. For test I altered the code to have a maximum of 6 lines, but when testing there were some slides that had 7 and even 8 lines. This is caused by the fact that only the last line of a verse can be moved to the next slide, so for long verses this does not work. This seems as a bug to me. See attached images.

To summarise, my questions are:
1. Is the fact that only the last line of a verse can be moved to the next slide a bug or a feature? If it is a bug I can try to fix it.
2. Can I add an option for the maximum number of lines instead of the hard coded characters per line divided by 4?

![image](https://user-images.githubusercontent.com/37149907/59183933-59744900-8b6d-11e9-8728-96e4be076f47.png)
![image](https://user-images.githubusercontent.com/37149907/59183982-6ee97300-8b6d-11e9-9de9-2c3f064797ee.png)

**Comment by berry120** [comment]
@jejbr84 Sorry, seem to have completely missed this! I think we've changed the way the bible text is shown reasonably significantly in the last couple of versions, is this still an issue you're having with 2021.0?
